c10/core/TensorImpl.h

```diff
@@ -361,31 +359,25 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
   }
 
   int64_t get_device() const {
```

Comment:
We should kill this entirely at some point. Not this PR though.

Reply:
do you mean to only have `device()` in TensorImpl?